### ISSUE TITLE
How to use Pull-to-Refresh like as ListView?

### ISSUE THREAD

**Issue by phamthanhtong** [roubachof/Xamarin-Forms-Practices#2]
HorizontalListView control did not have IsPullToRefreshEnabled property.
Could you share how to use Pull-to-Refresh like as ListView?

Thanks.

**Comment by roubachof** [comment]
Good question!

You can use the following layout:

```xml
<TaskLoaderView>
    <PullToRefreshLayout>
        <HorizontalListView>
```

It's the same here:

https://github.com/roubachof/Xamarin-Forms-Practices/blob/master/SillyCompany.Mobile.Practices/Presentation/Views/SillyDudePage.xaml

which is bound to:

https://github.com/roubachof/Xamarin-Forms-Practices/blob/master/SillyCompany.Mobile.Practices/Presentation/ViewModels/SillyDudeVm.cs

The view model is very simple: all the hard work is done by the ```ViewModelLoader``` and the ```TaskLoaderView```.

If you didn't know what is the ```TaskLoaderView```, have a look here:

https://github.com/roubachof/Sharpnado.Presentation.Forms#taskloaderview

**Comment by phamthanhtong** [comment]
I used your solution. Android is OK. But iOS is cannot pull to refresh.
`    <PullToRefreshLayout>
          <HorizontalListView>`
After that, I tried to change to:
`    <PullToRefreshLayout>
         <ScrollView>
             <HorizontalListView>`
iOS is OK. But Android cannot scroll up.

Could you give me a review?

**Comment by phamthanhtong** [comment]
@roubachof Could you share me a solution?

**Comment by roubachof** [comment]
@phamthanhtong if you send me a repo with a repro sample before thursday I will be able to have a look by the end of the week ;)

**Comment by roubachof** [comment]
Ok the issue with the RefreshView and the iOS version of the HorizontalListView will be fixed in v1.4.0
